src/pages/Jiyoung/components/Button/Button.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Button.scss';
+
+const Button = (props) => {
+  const { btnValue, btnPrimary, btnLine, btnText, disabled } = props;
+  return (
+    <>
+      {/* 
+      버튼 공통
+      - btn-primary
+      - btn-line
+      - btn-text
+      - btn-block
+      - btn-inline
+      - btn width : 96, 120
+      - btn height : 40, 48, 50, 56
+    */}
+      <button
+        type="button"
+        className={`btn ${btnPrimary ? 'btnPrimary' : ''}${
+          btnLine ? 'btnLine' : ''
+        }${btnText ? 'btnText' : ''}`}
```

Comment:
해당 조건을 만족하는 변수를 따로 선언해서 바로 작성해주는게 가독성 측면에서 더 도움이 될 것 같습니다
```suggestion
       const _btnPrimary = btnPrimary ? "btnPrimary" : ""
       const _btnLine = btnLine ? 'btnLine' : ''
       const _btnText = btnText ? 'btnText' : ' '
        className={`btn ${_btnPrimary} ${_btnLine} ${_btnText}`}
```